src/fusor/translator.py

```diff
@@ -50,10 +61,10 @@ def __init__(self, fusor: FUSOR) -> None:
 
     def _format_fusion(
         self,
-        gene_5prime: GeneElement,
-        gene_3prime: GeneElement,
-        tr_5prime: TranscriptSegmentElement | None = None,
-        tr_3prime: TranscriptSegmentElement | None = None,
+        gene_5prime: GeneElement | UnknownGeneElement,
+        gene_3prime: GeneElement | UnknownGeneElement,
+        tr_5prime: TranscriptSegmentElement | list[None] = None,
+        tr_3prime: TranscriptSegmentElement | list[None] = None,
```

Comment:
I still don't think this is what we want... In this method, you're accessing the first element in `tr_5prime` and `tr_3prime`, which aren't always `list[None]` (I don't think this makes sense either).